docs/source/tools/sandbox.rst

```diff
@@ -10,13 +10,15 @@ The DAML Sandbox, or Sandbox for short, is a simple ledger implementation that e
 
 You can start Sandbox together with :doc:`Navigator </tools/navigator/index>` using the ``daml start`` command in a DAML SDK project. This command will compile the DAML file and its dependencies as specified in the ``daml.yaml``. It will then launch Sandbox passing the just obtained DAR packages. Sandbox will also be given the name of the startup scenario specified in the project's ``daml.yaml``. Finally, it launches the navigator connecting it to the running Sandbox.
 
-It is possible to execute the Sandbox launching step in isolation by typing ``daml sandbox``.
+It is possible to execute the Sandbox launching step in isolation by typing ``daml sandbox --wall-clock-time``.
+
+Note: Sandbox is currently in the process of moving from Static Time mode to Wall Clock Time mode as the default. For this version only, you need to specify the time mode explicitly when launching Sandbox outside ``daml start``. (When you use ``daml start``, it provides this flag and any other flags in the ``sandbox-options:`` section of ``daml.yaml``.) From the next release, you can omit this flag.
 
 Sandbox can also be run manually as in this example:
 
 .. code-block:: none
 
-  $ daml sandbox Main.dar --scenario Main:example
+  $ daml sandbox Main.dar --static-time --scenario Main:example
```

Comment:
Any reason why we want static time here?

Reply:
Scenarios can't be run under wall clock time. (TBH we should probably just remove this section and tell people to use DAML Script.) It's explained on line 30.

Reply:
Oh of course, I completely missed the fact that this uses a scenario.

Reply:
I'd say let's remove everything about scenarios and add a link to DAML Triggers for initializing the ledger. Sorry to have missed this bit of text when deprecating scenarios the last time around.